tests/python/sealapi/main.py

```diff
@@ -0,0 +1,6 @@
+import sys
+import os
+import pytest
+
+if __name__ == "__main__":
+    sys.exit(pytest.main(["-s", "-v", "-x", os.path.dirname(os.path.abspath(__file__))]))
```

Comment:
Maybe comment what this is doing and specify that it's needed specifically by Bazel